R/pkg/DESCRIPTION

```diff
@@ -13,6 +13,7 @@ Authors@R: c(person("Shivaram", "Venkataraman", role = c("aut", "cre"),
 License: Apache License (== 2.0)
 URL: http://www.apache.org/ http://spark.apache.org/
 BugReports: http://spark.apache.org/contributing.html
+OS_Type: unix
```

Comment:
Can we do this in a separate PR ? It'll be good to track this as a separate JIRA / PR just to lookup why we did this. Also I'll hopefully hear from the winbuilder maintainer if this is something we can avoid doing.